android/java/res/layout/fragment_recovery_phrase.xml

```diff
@@ -1,149 +1,115 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.core.widget.NestedScrollView
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/recover_wallet_root"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/wallet_bg"
     android:scrollbars="vertical"
     android:fadeScrollbars="false"
     android:fillViewport="true">
 
-    <LinearLayout
+    <RelativeLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:gravity="center_vertical"
         android:orientation="vertical">
 
         <TextView
+            android:id="@+id/recovery_phrase_title"
+            style="@style/HeadingH3"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_alignParentTop="true"
+            android:layout_alignParentStart="true"
             android:gravity="center_horizontal"
-            android:layout_marginTop="32dp"
-            android:layout_marginStart="32dp"
-            android:layout_marginEnd="32dp"
-            android:text="@string/your_recovery_phrase"
+            android:layout_marginHorizontal="16dp"
+            android:layout_marginTop="14dp"
+            android:layout_marginBottom="18dp"
+            android:text="@string/recovery_phrase_title"
             android:textColor="@color/wallet_text_color"
-            android:textSize="18sp"/>
+            android:textStyle="bold"/>
```

Comment:
can we avoid using textStyle here ? it should be handled with font weight.

Reply:
@deeppandya , I went through all the places where `bold` is used. Would it be possible to create the bold equivalent for these styles? 

`Heading1` -> `Heading1Bold`
`Heading3` -> `Heading3Bold`
`DefaultRegular` -> `DefaultBold`
`XSmallSemibold` -> `XSmallBold`

Each of the new styles above can inherit all the attributes from their non-bold parent and just add `android:textStyle="bold"`. 
I can help with this if you want, then we merge, rebase this branch on top of master and remove all the bold attributes from all the components. How does that sound?

Reply:
@aguscruiz WDYT ? bold style could be handled with font weights.